src/shared/services/path-util.service.ts

```diff
@@ -26,6 +26,11 @@ import { Injectable } from '@angular/core';
 @Injectable()
 export class PathUtilService {
 
+  // to make sure it's constant/final
```

Comment:
Seperator string is a literal which is returned by a get method, insted of simple making it simple property (`seperator  = '/';`).

This ensures it can be never change during application lifetime which means it is a `constant`in other Java terminology `final` property. But if you think it's already clear we could remove the comment.